packages/terra-form-select/src/shared/_Dropdown.module.scss

```diff
@@ -20,4 +20,10 @@
     border-top-right-radius: var(--terra-form-select-dropdown-border-top-right-radius, 3px);
     box-shadow: var(--terra-form-select-dropdown-above-box-shadow, 0 -3px 12px -5px rgba(0, 0, 0, 0.25));
   }
+
+  .semantic {
+    position: absolute;
+    top: 100%;
```

Comment:
I'd recommend going with these styles:
```scss
    // 0 needs px unit for calc to work correctly
    /* stylelint-disable length-zero-no-unit */
    left: calc(0px - var(--terra-form-select-dropdown-left-offset-width, 1px));
    /* stylelint-enable length-zero-no-unit */
    top: calc(100% + var(--terra-form-select-dropdown-top-offset-width, 1px));
```

The `--terra-form-select-dropdown-left-offset-width` and `--terra-form-select-dropdown-top-offset-width` should be equal to the width of the border used in `--terra-form-select-dropdown-border`.  We'll also want to test this in RTL, we may need to define a right offset for that directionality.

Reply:
Earlier this morning when I initially added those, it shifted everything by 1px. Lemme see again, just to make sure..

Reply:
Talked offline, planning to go with the styling as-is.

Reply:
Added left: 0; to account for IE needing explicit style defined for positioning.